src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -72,66 +72,62 @@ public static class RSessionEvaluationCommands {
             return evaluation.RespondAsync(script);
         }
 
-        public static Task<REvaluationResult> PlotHistoryInfo(this IRSessionEvaluation evaluation) {
+        public static Task<REvaluationResult> PlotHistoryInfo(this IRExpressionEvaluator evaluation) {
             var script = @"rtvs:::graphics.ide.historyinfo()";
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
         }
 
-        public static Task<REvaluationResult> InstalledPackages(this IRSessionEvaluation evaluation) {
+        public static Task<REvaluationResult> InstalledPackages(this IRExpressionEvaluator evaluation) {
             var script = @"rtvs:::packages.installed()";
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
         }
 
-        public static Task<REvaluationResult> AvailablePackages(this IRSessionEvaluation evaluation) {
+        public static Task<REvaluationResult> AvailablePackages(this IRExpressionEvaluator evaluation) {
             var script = @"rtvs:::packages.available()";
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
         }
 
-        public static Task<REvaluationResult> InstallPackage(this IRSessionEvaluation evaluation, string packageName) {
+        public static Task<REvaluationResult> InstallPackage(this IRExpressionEvaluator evaluation, string packageName) {
             var script = string.Format("install.packages(\"{0}\")", packageName);
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
```

Comment:
And I think it's not `Json`, since it calls a built-in function? No-one calls it, so it's hard to tell.